onnx_mxnet/tests/test_layers.py

```diff
@@ -263,5 +263,12 @@ def test_squeeze(self):
         output = mxnet_backend.run_node(node_def, [ip1])[0]
         npt.assert_almost_equal(output, np.squeeze(ip1, axis=[1, 3]))
 
+    def test_reshape(self):
```

Comment:
For tests like this - we'd encourage you to contribute to the main ONNX repo instead and run it through onnx backend test infrastructure. That way the tests would be shared across multiple backends. See https://github.com/onnx/onnx/blob/master/docs/OnnxBackendTest.md @lupesko @anirudhacharya

Reply:
Yes, we are in the process of doing that. We plan on switching our CI process from running these tests to running all the tests through the ONNX backend test infrastructure. 

We are currently fixing a bunch of stuff on the import module so that all the operator conversions in our import module is completely in sync with the ONNX standards. 

Soon after we plan to add support for more operators, and for each of those operators we will add corresponding tests on the ONNX repo.

Reshape already has tests in the ONNX repo. This PR in fact fixes the failures we were getting from running the very samee tests.